schemas/context/identity.schema.json

```diff
@@ -40,5 +40,5 @@
       "$ref": "#/definitions/identity"
     }
   ],
-  "meta:status": "experimental"
+  "meta:status": "stabilizing"
```

Comment:
I think #203 is still in flight and might affect this. 👎

Reply:
👎 Not only that, but there are Adobe-centric references in here.